code_review_checklist.adoc

```diff
@@ -0,0 +1,46 @@
+# Pull Request review checklist for committers and reviewers
+:toc: auto
+
+The reviewer validates the content of a Pull Request with this checklist.
+
+## Automated validation steps
+
+These automated steps are blocking pull request merge.
```

Comment:
These automated steps are blocking pull request merge. > These automated steps block the pull request merge.

Reply:
```suggestion
These automated steps block the pull request merge.
```